examples/samplers/DEMetropolisZ_EfficiencyComparison.ipynb

```diff
@@ -4,15 +4,13 @@
    "cell_type": "markdown",
```

Comment:
<p>I wondered about the speed you got for some of these samplings - Python 3.11 might explain it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Plus an old windows machine?  </p><p> </p>